### PR TITLE
Add MEMBER_REPRESENTATION.md

### PR DESCRIPTION
**Pull request by christian-bromann**
As discussed in #67 this patch adds a markdown file to document team members involvement in standard organisations.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ The following groups automatically qualify for membership and can request to be
 * OpenJS Foundation CPC Members
 * OpenJS Foundation Project Maintainers
 
+Every team member is asked to document ones involvement in any of the standards organisation that person is regularly involved with.
```
Comment (ljharb) [inline GOVERNANCE.md:28]:
```suggestion
Every team member is asked to document ones involvement in any of the standards organizations that person is regularly involved with.
```

File: MEMBER_REPRESENTATION.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standard organisations:
```
Comment (ljharb) [inline MEMBER_REPRESENTATION.md:3]:
```suggestion
The following document lists OpenJS Foundation members and their representation to any standards organizations:
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -25,6 +25,8 @@ The following groups automatically qualify for membership and can request to be
 * OpenJS Foundation CPC Members
 * OpenJS Foundation Project Maintainers
 
+Every team member is asked to document ones involvement in any of the standards organizations that person is regularly involved with.
```
Comment (sendilkumarn) [inline GOVERNANCE.md:28]:
```suggestion
Every team member is asked to document one's involvement in any of the standards organizations that person is regularly involved with.
```

Comment (sendilkumarn) [reply GOVERNANCE.md:28]: Does it mean that we are restricting the regular members to be a member of an already existing standards org?

Comment (ljharb) [reply GOVERNANCE.md:28]: It says document involvement, not that involvement is required.

File: MEMBER_REPRESENTATION.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standards organizations:
+
+* <firstname> <lastname>
+    * [TC39][1]
+    * ...
+
+[1]: https://github.com/tc39
```
Comment (devsnek) [inline MEMBER_REPRESENTATION.md:9]:
```suggestion
[TC39]: https://github.com/tc39
```

File: MEMBER_REPRESENTATION.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standards organizations:
+
+* <firstname> <lastname>
```
Comment (devsnek) [inline MEMBER_REPRESENTATION.md:5]: richard and i are delegates to tc39 already, should we be added here now or do separate prs?

File: MEMBER_REPRESENTATION.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+    * [TC39][1]
```
Comment (devsnek) [inline MEMBER_REPRESENTATION.md:6]:
```suggestion
    * [TC39][]
```

File: GOVERNANCE.md [hunk 7]
```diff
@@ -25,6 +25,8 @@ The following groups automatically qualify for membership and can request to be
 * OpenJS Foundation CPC Members
 * OpenJS Foundation Project Maintainers
 
+Every team member is asked to document one's involvement in any of the standards organizations that person is regularly involved with.
```
Comment (MylesBorins) [inline GOVERNANCE.md:28]: I disagree with this framing. I was imagining that we would be documenting who is representing openjs at these meetings.

I participate in a number of standards bodies in my role at Google and don't think it would be appropriate, or even helpful, to document them all here.

Further, I don't think it is a requirement to have to be a member of this working group to have your name included in the list.

I think this paragraph should be removed

File: MEMBER_REPRESENTATION.md (hunk 5 above)
Comment (christian-bromann) [reply MEMBER_REPRESENTATION.md:5]: Let's do separate PRs for adding individuals

File: GOVERNANCE.md (hunk 7 above)
Comment (christian-bromann) [reply GOVERNANCE.md:28]: @MylesBorins makes sense, I will remove it

File: MEMBER_REPRESENTATION.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standards organizations:
+
+* <firstname> <lastname>
+    * [TC39][]
+    * ...
```
Comment (eemeli) [inline MEMBER_REPRESENTATION.md:7]:
```suggestion
* [TC39]
  * <firstname> <lastname> (#issue)
  * ...
```

I think the indexing is the wrong way around here, and rather than focusing on which members are representing us, we should focus on which organisations we are represented in. This would also help clarify that we're enumerating representations of OpenJS in particular, not all representations by members of the Working Group.

I also added an `#issue` link to the template, for the issue requesting and approving the representation.

File: MEMBER_REPRESENTATION.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standards organizations:
+
+* [TC39]
+  * <firstname> <lastname> (#issue)
```
Comment (mikesamuel) [inline MEMBER_REPRESENTATION.md:6]: `<firstname> <lastname>` is a poor fit for names from many cultures.

Maybe `<fullname>`

I believe that term is used in vCard to refer to a non-abbreviated name.

**Comment by christian-bromann** [comment]
@eemeli any outstanding concerns?

**Comment by eemeli** [comment]
Merging, as there are no more outstanding concerns, there is support from multiple members, and the PR has been open for well over 72 hours.
